Source/items.h

```diff
@@ -170,6 +170,15 @@ enum icreateinfo_flag2 {
 	// clang-format on
 };
 
+enum selection_flag : uint8_t {
+	// clang-format off
+	SF_NONE,
+	SF_FLOOR,
+	SF_ELEVATED,
+	SF_ALL,
+	// clang-format on

```

Comment:
why are you disabling clang format here? the previous enum declaration does that to keep the assignment/values inline and preserve the gap between the individual flags and the group flags, shouldn't be necessary here.